RELEASE_NOTES.md

```diff
@@ -9,14 +9,4 @@
 [//]: # ( - Configurations public constructor are deprecated, please use each Configuration's builder to make a Configuration object)
 
 ## Added
-- Validation of PublicKey in CSE module if it's used standalone.
-- 'isReady' flag to ComponentState because some components might require some initialization time even if all the inputs are valid.
-- 'isValid' now checks both 'isInputValid' and 'isReady' to be true.
-
-## Fixed
-- Handle Intent results if DropInActivity got destroyed.
-- Queue API request if DropInService is not yet bound to DropInActivity
-
-
-
-
+- Return `returnUrlQueryString` from redirect URL.
```

Comment:
Maybe it's nice to add "for some redirect payment methods like Swish", otherwise this change doesn't mean anything to the merchants as this is mostly an internal change.

Reply:
Or is it better to remove this completely from the release notes?